snap/snapcraft.yaml

```diff
@@ -27,6 +27,22 @@ slots:
     source:
       write:
         - "$SNAP_COMMON/run/switch"
+  ovn:
```

Comment:
This should be probably named `ovn-central` to match naming convention of already existing `ovn-chassis`, which also exposes runtime directory.